cumulusci/tasks/apex/testrunner.py

```diff
@@ -265,19 +273,18 @@ def _get_test_results(self):
             self.logger.error("-" * 80)
             counter = 0
             for result in test_results:
-                if result["Outcome"] not in ["Fail", "CompileFail"]:
-                    continue
```

Comment:
... oh wow.

Reply:
The logic is the same as before (I inverted the condition and moved the code that was outside it to be inside) but this got me out of adding a second test case to the fixture just to get line coverage here.

Reply:
its smart!